packages/web-components/src/tabs/tab-list/tab-list.ts

```diff
@@ -0,0 +1,82 @@
+import { attr, booleanConverter } from '@microsoft/fast-element';
+import { FASTTabs } from '@microsoft/fast-foundation';
+
+export interface TabData {
+  id: string;
+  x: number;
+  y: number;
+  height: number;
+  width: number;
+}
+/**
+ * TabList extends FASTTabs and is used for constructing a fluent-tab-list custom html element.
+ *
+ * @class TabList component
+ * @public
+ */
+export class TabList extends FASTTabs {
+  @attr appearance?: 'subtle' | 'transparent';
+
+  @attr({ converter: booleanConverter })
+  disabled?: boolean;
```

Comment:
You can use `mode: "boolean"` instead of importing and including that converter.